notebooks/GPU.ipynb

```diff
@@ -36,7 +36,7 @@
       "outputs": [],
       "source": [
         "#@title Select your model below, then click the play button to start the UI.\n",
-        "#@markdown Afterwards, just sit tight and wait - the link to the UI should show up after it's done starting up.\n",
+        "#@markdown Afterwards, just sit tight and wait - the link to the UI should show up after it's done starting up.\nWARNING: The 6B Experimental model will currently produce broken outputs due to a bug in Kobold. We highly recommend you use the standard Pygmalion 6B model until it is fixed.\n",
```

Comment:
Also, instead of "due to a bug in Kobold" I'd say "due to a bug _in our Kobold integration_" - theoretically Kobold banning EOS isn't a bug but an intended feature. Just one we didn't expect, and that doesn't play nice with our SFT models.